srcs/django_code/front/js/helpers.js

```diff
@@ -0,0 +1,28 @@
+export const translate = async (key) => {
+    const default_lang = "en"
+    let lang = await cookieStore.get("lang")
+
+    if (lang == null)
+    {
+        cookieStore.set({ name: "lang", value: default_lang })
+        lang = { value: default_lang }
+    }
+
+    const response = await fetch(`https://${location.hostname}:${location.port}/static/lang/${lang.value}.json`)
+
+    if (response.ok)
+    {
+        const value = (await response.json())[key]
+
+        if (value == null)
+        {
+            cookieStore.set({ name: "lang", value: default_lang })
+            return await translate(key)
```

Comment:
Eviter les boucle infinie par un set par deffault

```suggestion
            value = "NULL"
```

Reply:
Fix ici [86c995b](https://github.com/Rreyth/get-transcended/pull/61/commits/86c995b51a72bdb4e9b062e397628af941ead2ee)